packages/sacado/src/Kokkos_DynRankView_Fad.hpp

```diff
@@ -907,14 +907,14 @@ void deep_copy
 //         dst.span_is_contiguous() && //will always fail - stride set to 0s
 //         src.span_is_contiguous() &&
          dst.span() == src.span() &&
-         dst.dimension_0() == src.dimension_0() &&
-         dst.dimension_1() == src.dimension_1() &&
-         dst.dimension_2() == src.dimension_2() &&
-         dst.dimension_3() == src.dimension_3() &&
-         dst.dimension_4() == src.dimension_4() &&
-         dst.dimension_5() == src.dimension_5() &&
-         dst.dimension_6() == src.dimension_6() &&
-         dst.dimension_7() == src.dimension_7() ) {
+         dst.extent(0) == src.dimension_0() &&
```

Comment:
Did you miss this `dimension_0()`, or is that one of those ViewOffset things?

Reply:
Nope, I missed that one.  Good catch!